pkg/tracer/tcptracer-ebpf.go

```diff
@@ -0,0 +1,235 @@
+// Code generated by go-bindata.
```

Comment:
It changes `_tcptracerEbpfO` and the size in

```
info := bindataFileInfo{name: "tcptracer-ebpf.o", size: 18592, mode: os.FileMode(420), modTime: time.Unix(1, 0)}
```

https://github.com/weaveworks/tcptracer-bpf/pull/4/files#diff-64d61ca413c763faec03094f7b804a77R86